scipy/stats/_multivariate.py

```diff
@@ -158,10 +158,13 @@ def __init__(self, M, cond=None, rcond=None, lower=True,
 
         eps = _eigvalsh_to_eps(s, cond, rcond)
         if np.min(s) < -eps:
-            raise ValueError('the input matrix must be positive semidefinite')
+            msg = "The input matrix must be symmetric positive semidefinite."
+            raise ValueError(msg)
         d = s[s > eps]
         if len(d) < len(s) and not allow_singular:
-            raise np.linalg.LinAlgError('singular matrix')
+            msg = ("When `allow_singular is False`, the input matrix must be "
+                  "symmetric positive definite.")
```

Comment:
```suggestion
            msg = ("When `allow_singular is False`, the input matrix must be "
                   "symmetric positive definite.")
```

Reply:
I've seen you recommend this beforem. Where does it come from? (I mean is it in PEP8 or something? I don't think a space is a binary operator. I get that it's slightly easier to verify that the space wasn't forgotten.)

Reply:
Here it's to maintain the visual alignment. Cf the lint failure `scipy/stats/_multivariate.py:166:19: E128 continuation line under-indented for visual indent`

Reply:
Oh I am looking on my phone and I got it confused with a space after the quotation. Have I seen you recommended that before?